src/main/java/Duke.java

```diff
@@ -1,10 +1,47 @@
+import duke.command.Command;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.util.TaskList;
+
+/**
+ * Represents a Personal Assistant Chatbot that helps a person to keep track of various tasks.
+ */
```

Comment:
Good job on this, I wasn't so on what comment to write for this but you gave me a really good idea 👍